packages/financial-templates-lib/test/truffle/UniswapPriceFeed.js

```diff
@@ -155,6 +155,51 @@ contract("UniswapPriceFeed.js", function(accounts) {
     assert.equal(uniswapPriceFeed.getCurrentPrice(), null);
   });
 
+  it("One event within window, several before", async function() {
```

Comment:
Same comment as [above](https://github.com/UMAprotocol/protocol/pull/2426/files#r567926321)

Reply:
So is it correct to say that the latest data point before the window is used in the TWAP until the first data point within the window?

Reply:
Yep, that's correct.